feature-libs/order/components/order-history/order-history-extended/consignment-entries/consignment-entries.component.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Consignment } from '@spartacus/order/root';
+
+@Component({
+  selector: 'cx-consignment-entries',
+  templateUrl: './consignment-entries.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConsignmentEntriesComponent {
+  @Input()
+  consignments?: Consignment[];
+  @Input()
+  orderCode?: string;
+
+  consignmentEntriesLength(consignment: Consignment): number {
```

Comment:
[MEDIUM]
For consistency, let's stick to a convention of starting names of methods with verbs, e.g. `getConsignmentEntriesLength()`

Note: same remark applies to other places in this PR

Reply:
removing this method, as its not used